pkg/certificate/certificate.go

```diff
@@ -211,6 +211,8 @@ type Request struct {
 	Location      *Location
 	ValidityHours int
 	IssuerHint    string
+	// PATCH: ValidityDuration is an improved version of ValidityHours.
+	ValidityDuration *time.Duration
```

Comment:
Expand this comment, explaining the improvements and explaining how it interacts with ValidityHours.